README.md

```diff
@@ -149,6 +149,21 @@ An example project on how to upgrade from older tags can be found in the Integri
 
 If you still experience issues during upgrading, do not hesitate to create an issue for support.
 
+## Version numbers
+
+We use a specififc versioning schema in order to enable us to release the same version multiple times for different polkadot releases
```

Comment:
I would change this line to
Starting with release 1.16.0, we use a specific versioning schema...

Reply:
👍 I changed it as you suggested